cmd/soroban-rpc/internal/events/events.go

```diff
@@ -1,6 +1,8 @@
+//nolint:mnd
```

Comment:
Why was this removed? We shouldn't be using magic numbers in production code.

Reply:
I removed this because of this line:

```
Objectives: map[float64]float64{0.5: 0.05, 0.9: 0.01, 0.99: 0.001}
```

Do I then need to define constants for each key-value pair here?

Reply:
Anyways, I have fixed it now. I agree though - in general its good to not have any magic numbers defined anywhere.